tokio/src/runtime/queue.rs

```diff
@@ -345,13 +358,16 @@ impl<T> Steal<T> {
             let n = src_tail.wrapping_sub(src_head_real);
             let n = n - n / 2;
 
+            assert!(n <= 128, "actual = {}", n);
```

Comment:
nit: does it make sense to express this as 
```suggestion
            assert!(n <= LOCAL_QUEUE_CAPACITY / 2, "actual = {}", n);
```
because the queue capacity is set much lower under loom?

Reply:
huh, per conversation on Discord, I believe @jebrosen is now hitting _this_ assertion...

Reply:
This was the wrong place to put it. I placed it outside the CAS loop.